food/ios/App/FilterViewController.swift

```diff
@@ -62,7 +62,6 @@ class FilterViewController: UIViewController {
     if self.fourDollars.isSelected {
       prices.append(4)
```

Comment:
Rating maximum value is 4: https://food-stag.lenfestlab.org/admin/post?model_name=post&utf8=%E2%9C%93&f%5Brating%5D%5B16817%5D%5Bo%5D=default&f%5Brating%5D%5B16817%5D%5Bv%5D%5B%5D=5&f%5Brating%5D%5B16817%5D%5Bv%5D%5B%5D=&f%5Brating%5D%5B16817%5D%5Bv%5D%5B%5D=&query=